src/renderer/style/buttons.scss

```diff
@@ -0,0 +1,28 @@
+.button {
```

Comment:
should we put this in /components/buttons?

Reply:
Yes, seems better and will have more buttons anyway.